kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-case-mgmt/kie-wb-common-stunner-case-mgmt-client/src/main/java/org/kie/workbench/common/stunner/cm/client/wires/VerticalStackLayoutManager.java

```diff
@@ -72,7 +73,17 @@ public void layout(final WiresContainer container) {
         for (WiresShape ws : container.getChildShapes()) {
             ws.setLocation(new Point2D(PADDING_X,
                                        y));
-            y = y + ws.getPath().getBoundingBox().getHeight() + PADDING_Y;
+            y = y + getTotalHeight(ws) + PADDING_Y;
```

Comment:
Hi @jeremyary, I added another commit to the pull request.
When drag/drop a stage to a parent stage, the child stage is always placed as the first child not the last.
When adding multiple stages with child elements to a parent stage, the child elements overlap.
This change is to fix these issues.